client/testing/test_quickstart.py

```diff
@@ -10,7 +10,7 @@ def test_dryrun(cmd_devpi):
 
 @pytest.mark.skipif("config.option.fast")
 def test_functional(cmd_devpi, monkeypatch, tmpdir):
-    monkeypatch.setenv("DEVPI_SERVERDIR", tmpdir.join("server"))
+    monkeypatch.setenv("DEVPISERVER_SERVERDIR", tmpdir.join("server"))
     monkeypatch.setenv("DEVPI_CLIENTDIR", tmpdir.join("client"))
```

Comment:
why is this not changed to ``DEVPICLIENT_CLIENTDIR``?

Reply:
This PR only adds configuration file support to devpi-server. If we ever add it to devpi-client, then it should be updated there.